### ISSUE TITLE
Buttons in the Notification Overview

### ISSUE THREAD

**Issue by Jonas2OO3** [Waboodoo/HTTP-Shortcuts#244]
**Is your feature request related to a problem? Please describe.**
Controlling the lights in our apartment, would like to reach those functions more quickly and from a locked phone.

**Describe the solution you'd like**
Being able to add buttons/shortcuts to the notification overview / pull down / ... not sure what its called ...?

**Describe alternatives you've considered**
No idea.

**Additional context**
For example, Spotify adds prev/pause/next buttons in its notification when running.

**Comment by Jonas2OO3** [comment]
Hi,

thanks, I didn't know about that function. Would it be possible to allow triggering shortcuts without unlocking the phone (possibly configurable globally or for each shortcut separately)?

Awesome app btw.

**Comment by Waboodoo** [comment]
I've given this some thought and decided to close this issue now, as no one has requested this feature in the last 3 years. 
Adding a permanently visible notification for this seems too obtrusive to me, and I believe this use case is now sufficiently covered by the recently added "Device Control" support and by the Quick Settings Tile support.

**Comment by Jonas2OO3** [comment]
Yes, thank you, the Quick Settings Tile Support sufficiently covers this.

One final question though: is it possible to (change a setting to allow to) trigger the shortcut without unlocking the phone?

**Comment by Waboodoo** [comment]
> One final question though: is it possible to (change a setting to allow to) trigger the shortcut without unlocking the phone?

It depends a bit on your phone (Android version, launcher, etc.), but in general, it should be possible to trigger a shortcut even when the phone is locked. However, it won't be able to display any UI, so it would only really work if your shortcut doesn't rely on any variables and scripting that opens dialogs or other windows.
It might also help to enable the "Run in background" setting in the shortcut editor's "Trigger & Execution Settings" section.
